keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/UnhappyVetoRepository.kt

```diff
@@ -55,11 +55,9 @@ abstract class UnhappyVetoRepository(
    *
    * @param wait the time to wait before re-checking the resource, `null` means "never re-check".
    */
-  abstract fun getOrCreateVetoStatus(
-    resourceId: String,
-    application: String,
-    wait: Duration? = Duration.parse(waitingTime)
-  ): UnhappyVetoStatus
+  abstract fun getRecheckTime(
```

Comment:
Refactored this to do just the repository part, which is to return data from the database.